### DISCUSSION TITLE
Tilt strokes on argon spinner depending on spinning speed

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#20427] [Ideas]
I mean keep them vertical (like in triangles one) if you don't spin it; and tilt to diagonal (like now) depending on SPM. This may improve spinners' feedback.

**Comment by frenzibyte** [comment]
By strokes you mean the spinner ticks? and you want them to tilt based on the SPM?

**Reply by Feodor0090** [reply]
I'm talking about these things.
![изображение](https://user-images.githubusercontent.com/53872073/191971849-7b6d05d4-7015-4f4d-9ce2-656a7b30ffcd.png)
My proposal is to make their angle depending on SPM, so they stay vertical on 0 SPM, like now on 477 SPM (AT default), and more horizontal on higher SPMs.

If they would be anchored on the outer side, this may create feeling of centrifugal force.

**Comment by frenzibyte** [comment]
https://user-images.githubusercontent.com/22781491/192406942-942a3950-9aee-4360-a2c8-e10467ff0085.mp4

Seems like it may work well, @peppy @arflyte thoughts?

**Reply by peppy** [reply]
I'd want to test it in actual gameplay, hard to say from autoplay. I think it has potential though.
